azure/package.json

```diff
@@ -11,13 +11,30 @@
   "license": "MIT",
   "author": "Microsoft and contributors",
   "scripts": {
-    "build": "npm run build:genver && npm run build:compile && npm run lint",
+    "build": "npm run policy-check && npm run layer-check && npm run build:genver && npm run build:compile && npm run lint && npm run build:docs",
+    "build:ci": "npm run build:genver && lerna run build:compile --stream",
     "build:compile": "lerna run build:compile --stream",
+    "build:docs": "lerna run build:docs --stream --parallel",
+    "build:fast": "fluid-build --root ..",
+    "build:full": "npm run build:genver && npm run build:full:compile && npm run lint && npm run build:docs",
+    "build:full:compile": "lerna run build:full:compile --stream",
     "build:genver": "lerna run build:genver --stream --parallel",
-    "clean": "rimraf dist *.tsbuildinfo *.build.log",
+    "bump-version": "fluid-bump-version --root ..",
+    "bundle-analysis:collect": "npm run webpack:profile && fluid-collect-bundle-analyses",
+    "bundle-analysis:run": "fluid-run-bundle-analyses",
+    "ci:build": "npm run build:genver && lerna run build:compile --stream",
+    "ci:build:docs": "lerna run ci:build:docs --stream --parallel",
+    "ci:test": "npm run test:report ; t1=$? ; npm run test:copyresults ; exit $t1",
+    "ci:test:coverage": "npm run test:coverage ; t1=$? ; npm run test:copyresults ; exit $t1",
+    "clean": "lerna run clean --stream --parallel && npm run clean:docs && npm run clean:nyc",
+    "clean:docs": "rimraf **/_api-extractor-temp docs/api/*/**",
+    "clean:nyc": "rimraf nyc/**",
+    "clean:r11s": "cd server/routerlicious && npm run clean",
     "postinstall": "npm run postinstall:lerna",
-    "lint": "lerna run lint --no-sort --stream",
-    "lint:fix": "lerna run lint:fix --no-sort --stream",
+    "layer-check": "fluid-layer-check --info ../tools/build-tools/data/layerInfo.json",
+    "lerna": "lerna",
+    "lint": "lerna run lint --no-sort --stream -- -- -- --color",
+    "lint:fix": "lerna run lint:fix --no-sort --stream -- -- -- --color",
```

Comment:
Double checking all of these scripts will run correctly within the azure folder.

Reply:
yup